### PR TITLE
Add the issue and pull request templates

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**
This PR adds the pull request and issue templates to improve the contributing experience.

**Related issues**
https://github.com/0xAX/asm/issues/22

### CODE REVIEW

File: .github/issue-template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- Thank you for your contribution. Before you submit the issue:
+- Search open and closed issues for duplicates.
+- Read the [Contributing guidelines](../contributing.md).
+- Follow the [Code of Conduct](../CODE_OF_CONDUCT.md).
```
Comment (klaudiagrz) [inline .github/issue-template.md:4]: Not about such links here. Maybe we should skip the links?

Comment (0xAX) [reply .github/issue-template.md:4]: Why?

**Review by 0xAX — APPROVED** [review]

File: .github/issue-template.md (hunk 1 above)
Comment (klaudiagrz) [reply .github/issue-template.md:4]: They are not clickable anyway in this form 🤔

Comment (0xAX) [reply .github/issue-template.md:4]: Ah it is comment. I see what you mean. I'd leave it as is anyway. Maybe someone will pay attention.

**Pull request merged**